app/javascript/components/button/button-component.jsx

```diff
@@ -68,7 +71,18 @@ const Button = props => {
   }
 
   if (tooltip) {
-    return <Tooltip {...tooltip}>{button}</Tooltip>;
+    return (
+      <Tooltip
+        theme="tip"
+        position="top"
+        arrow
+        disabled={isDeviceTouch}
+        html={<Tip text={tooltip.text} />}
```

Comment:
Yeah, probably we will never use other component instead of `<Tip>` and if this happens we can overwrite it with `{...tooltip}`. I love it! good job.